app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/individual/IndividualHealthRecordViewModel.kt

```diff
@@ -149,8 +152,28 @@ class IndividualHealthRecordViewModel @Inject constructor(
         return BcscInfo(dto, isAuthenticatedPatientAvailable, isBcscSessionActive)
     }
 
-    fun executeOneTimeDataFetch() {
-        workerInvoker.executeOneTimeDataFetch()
+    fun executeOneTimeDataFetch() = viewModelScope.launch {
+        try {
+            mobileConfigRepository.refreshMobileConfiguration()
```

Comment:
We don't need this call since the worker below is already calling the mobileConfiguration

Reply:
It is required, as worker can not pass the exception to the fragment & we are not able to display the error message in the snack bar

Reply:
decided to go with current implementation & will revisit the implementation, as it is non trivial